mirar/pipelines/winter/config/files/avro_schema/winter.alert.avsc

```diff
@@ -6,14 +6,14 @@
     "fields": [
         {"name": "schemavsn", "type": "string", "doc": "schema version used", "default": "0.1"},
         {"name": "publisher", "type": "string", "doc": "origin of alert packet", "default": "WINTER-mirar"},
-        {"name": "objectId", "type": "string"},
+        {"name": "objectid", "type": "string"},
         {"name": "candid", "type": "long"},
         {"name": "candidate", "type": "winter.alert.candidate", "doc": "candidate information"},
         {"name": "prv_candidates", "type": [{
             "type": "array",
             "items": "winter.alert.prv_candidate"}, "null"]},
-        {"name": "cutoutScience", "type": "bytes", "doc": "science cutout image [gzipped fits bytestream]"},
-        {"name": "cutoutTemplate", "type": "bytes", "doc": "cutout of template image [gzipped fits bytestream]"},
-        {"name": "cutoutDifference", "type": "bytes", "doc": "difference image cutout [gzipped fits bytestream]"}
+        {"name": "cutout_science", "type": "bytes", "doc": "science cutout image [gzipped fits bytestream]"},
```

Comment:
This differs from ZTF? Is this going to cause problems in Fritz?

Reply:
We will need our own dedicated schema conversion code in any case, so I don't think this adds much additional complexity. We can revert if Theo complains?

Reply:
I would prefer to keep it this way if possible, since both python and avro use `snake_case`. So, unlike many other scenarios, our writer-language, reader-language and message format all agree/expect that variables should be named that way.

Reply:
I would prefer to keep it this way if possible, since python uses `snake_case`, and snake_case is exclusively used throughout the rest of our avro schema for every other field. So, our writer-language, reader-language and message format all agree/expect that variables should be named that way.